src/phoenix/trace/exporter.py

```diff
@@ -9,6 +9,8 @@
 
 import opentelemetry.proto.trace.v1.trace_pb2 as otlp
 import requests
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
```

Comment:
Let's not add more code to this exporter module, because the span related functionality is being deprecated and should eventually be removed.

Reply:
The functionality is very tightly isolated and easily moved when we need, I think adding this is okay, as we aren't exposing anything new